test/unit/org/apache/cassandra/ServerTestUtils.java

```diff
@@ -25,6 +25,10 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.apache.cassandra.auth.CassandraRoleManager;
```

Comment:
No changes needed to this file now?

Reply:
indeed, that would be caught by checkstyle anyway